services/Resources/subscriptions/Deploy-ActivityLog-ResourceHealth-UnHealthly-Alert.json

```diff
@@ -195,19 +187,18 @@
                   "enabled": {
                     "type": "string"
                   },
-                  "BYOActionGroup": {
-                    "type": "String"
+                  "BYOActionGroups": {
+                    "type": "Array"
                   }
                 },
                 "variables": {
-                  "varBYOActionGroups": "[[split(parameters('BYOActionGroup'), ',')]",
-                  "copy": [
+                  "copyLoop": [
                     {
-                      "name": "varActionGroupIds",
-                      "count": "[[length(variables('varBYOActionGroups'))]",
+                      "name": "actionGroups",
+                      "count": "[[[length(parameters('BYOActionGroups'))]",
```

Comment:
This is because we move the copy logic into another deployment scope, in order for arm schema to accept it, otherwise it will try to resolved the copy logic before needing to